djangosaml2idp/views.py

```diff
@@ -177,48 +225,36 @@ def get(self, request, *args, **kwargs):
         except Exception as excp:
             return self.handle_error(request, exception=excp)
 
-        # Signature verification
-        # for authn request signature_check is saml2.sigver.SecurityContext.correctly_signed_authn_request
-        verified_ok = req_info.signature_check(req_info.xmlstr)
-        if not verified_ok:
-            return self.handle_error(request, extra_message="Message signature verification failure", status=400)
-
-        # Gather response arguments
-        try:
-            resp_args = self.IDP.response_args(req_info.message)
-        except (UnknownPrincipal, UnsupportedBinding) as excp:
-            return self.handle_error(request, exception=excp, status=400)
+        # check SAML request signature
+        self.verify_request_signature(req_info)
```

Comment:
You're right, this method was created just as an delegation interface. Otherwise today I've tested pySAML2 internals about sign verification and I hae good news. Bytheway I'd like to hold another validation in our code, to avoids problem for future pySAML2 bugs (it have)